utils/core.php

```diff
@@ -35,7 +35,7 @@ function fetch($sql, ...$args)
     return query_db($sql, ...$args)->fetchAll();
 }
 /** Calls fetch() and throws if there isn't a single element.
- * @return array a single DB entity.
+ * @return entity a single DB entity.
```

Comment:
This is not a valid PHP type. See https://www.php.net/manual/fr/language.types.intro.php

Reply:
I know our database doesn't have models (yet). unfortunately, in the meantime, the best we can do is an anonymous `object` or an `array` type

Reply:
Ah oui je croyais que c'étais un nom. Je vais mettre `object` du coup.

Reply:
Ah ! Sauf si les `array` en PHP sont aussi des dictionnaires. Du coup ce que retourne la requête du PDO, ce sont des `array`s ?

Reply:
All PHP objects are dictionaries. Which means you can always access an array like an object and vice versa.

Depending on the syntax we prefer, we can change it around, especially if we get around creating type definitions for our models.

Side note: except if you really can't, I would appreciate keeping discussions in English so that this repo remains as internationally accessible as possible